tests/test_invalidation.py

```diff
@@ -1,28 +1,30 @@
 #!/usr/bin/env python
 # coding:utf-8
 from meshroom.core.graph import Graph
-from meshroom.core import desc, registerNodeType
+from meshroom.core import desc, pluginManager
+from meshroom.core.plugins import NodePlugin
 
 
 class SampleNode(desc.Node):
     """ Sample Node for unit testing """
     inputs = [
-        desc.File(name='input', label='Input', description='', value='',),
-        desc.StringParam(name='paramA', label='ParamA', description='', value='', invalidate=False)  # No impact on UID
+        desc.File(name="input", label="Input", description="", value="",),
+        desc.StringParam(name="paramA", label="ParamA",
+                         description="", value="",
+                         invalidate=False)  # No impact on UID
     ]
     outputs = [
         desc.File(name='output', label='Output', description='', value="{nodeCacheFolder}")
     ]
 
-
-registerNodeType(SampleNode)
-
+nodePlugin = NodePlugin(SampleNode)
+pluginManager.registerNode(nodePlugin)  # register standalone NodePlugin
```

Comment:
Use a new dedicated method `pluginManager.registerNodeDesc(SampleNode)`
and later unregisterNodeDesc()